.github/workflows/test.yml

```
@@ -0,0 +1,213 @@
+            else
+              echo "No tests found for adapter-appwrite"
+            fi
+          else
+            echo "adapter-appwrite package not found"
+          fi
+          
+      - name: Generate test coverage
+        run: |
+          cd packages/adapter-electron
+          pnpm run test:coverage
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)